app/Libs/ProductApiDriver.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: macintosh
+ * Date: 8/1/16
+ * Time: 1:20 PM
+ */
+
+namespace App\Libs;
+
+
+class ProductApiDriver
+{
+    public $apiUrl;
+    public $httpClient;
+
+    public function __construct() {
+        $this->apiUrl = env('CDS_PRODUCT_API_URL', '');
+        $this->httpClient = new \GuzzleHttp\Client([
+            'base_uri' => $this->apiUrl
+        ]);
+    }
+
+    private function getResponse($response) {
+        $content = $response->getBody()->getContents();
+        $resJson = json_decode($content);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$resJson". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1488)